xapian-letor/cmake/xapianletor-config.cmake.in

```diff
@@ -0,0 +1,5 @@
+SET(prefix "@prefix@")
+SET(exec_prefix "@exec_prefix@")
+SET(XAPIAN_LIBRARIES "@libdir@/libxapianletor@SHLIBEXT@" CACHE FILEPATH "Libraries for Xapian-letor")
+SET(XAPIAN_INCLUDE_DIR "@incdir@" CACHE PATH "Include path for Xapian-letor")
+SET(XAPIAN_FOUND "TRUE")
```

Comment:
I'm certainly not a cmake expert, but I think we probably need to use different names here to xapian-core (`XAPIANLETOR_LIBRARIES`, etc would seem the obvious choice.)

Reply:
fixed in a fixup commit.